### PR TITLE
Additional retry cmds

### PR DESCRIPTION
**Pull request by ar3ndt**

### CODE REVIEW

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh [hunk 1]
```diff
@@ -93,14 +93,15 @@ download_file() {
 
 	local file_name=$(basename "$file_url")
 	local dest_path="$dest_dir/$file_name"
+	local retries="3"
```
Comment (to-bar) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh]:
```suggestion
	local retries=3
```

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh [hunk 2]
```diff
@@ -114,6 +115,7 @@ download_image() {
 	local tag=${splited_image[1]}
 	local repo_basename=$(basename -- "$repository")
 	local dest_path="${dest_dir}/${repo_basename}-${tag}.tar"
+	local retries="3"
```
Comment (to-bar) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh]:
```suggestion
	local retries=3
```

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh [hunk 3]
```diff
@@ -134,10 +134,11 @@ download_packages() {
 	local dest_dir="$1"
 	shift
 	local packages="$@"
+	local retries="3"
```
Comment (to-bar) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh]:
```suggestion
	local retries=3
```

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/download-requirements.sh [hunk 4]
```diff
@@ -93,14 +93,15 @@ download_file() {
 
 	local file_name=$(basename "$file_url")
 	local dest_path="$dest_dir/$file_name"
+	local retries="3"
```
Comment (to-bar) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/download-requirements.sh]:
```suggestion
	local retries=3
```

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/download-requirements.sh [hunk 5]
```diff
@@ -114,6 +115,7 @@ download_image() {
 	local tag=${splited_image[1]}
 	local repo_basename=$(basename -- "$repository")
 	local dest_path="${dest_dir}/${repo_basename}-${tag}.tar"
+	local retries="3"
```
Comment (to-bar) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/download-requirements.sh]:
```suggestion
	local retries=3
```

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/download-requirements.sh [hunk 6]
```diff
@@ -134,10 +134,11 @@ download_packages() {
 	local dest_dir="$1"
 	shift
 	local packages="$@"
+	local retries="3"
```
Comment (to-bar) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/download-requirements.sh]:
```suggestion
	local retries=3
```

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/common.sh [hunk 7]
```diff
@@ -44,6 +44,7 @@ download_image() {
 	local tag=${splited_image[1]}
 	local repo_basename=$(basename -- "$repository")
 	local dst_image="${dest_dir}/${repo_basename}-${tag}.tar"
+	local retries="3"
```
Comment (to-bar) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/common.sh]:
```suggestion
	local retries=3
```

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/common.sh [hunk 8]
```diff
@@ -64,6 +65,7 @@ download_file() {
 
 	local file_name=$(basename "$file_url")
 	local dest_path="$dest_dir/$file_name"
+	local retries="3"
```
Comment (to-bar) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/common.sh]:
```suggestion
	local retries=3
```

**Review by to-bar — CHANGES_REQUESTED** [review]
Please add entry to changelog.

**Review by to-bar — APPROVED** [review]

**Review by atsikham — APPROVED** [review]

**Pull request merged**